core/arch/arm32/plat-vexpress/conf.mk

```diff
@@ -25,6 +25,15 @@ WITH_SECURE_TIME_SOURCE_CNTPCT := y
 WITH_UART_DRV := y
 WITH_GIC_DRV := y
 
+# SE API is only suppoorted by QEMU Virt platform
+ifeq ($(PLATFORM_FLAVOR),qemu_virt)
+WITH_SE_API := y
+WITH_SE_API_SELF_TEST := y
+WITH_PCSC_PASSTHRU_READER_DRV := y
+platform-cppflags += -DWITH_SE_API=1
+core-platform-cppflags += -DWITH_PCSC_PASSTHRU_READER_DRV=1
```

Comment:
I suggest you name your configuration variables with the CFG_ prefix so that they are automatically included in conf.h. For instance CFG_SE_API, CFG_SE_API_SELF_TEST, etc. Then you don't need to add them manually to the core CPP flags.